app/services/api/v1/authorize_api_request.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class Api::V1::AuthorizeApiRequest
+  prepend SimpleCommand
+
+  def initialize(headers = {})
+    @headers = headers
+  end
+
+  def call
+    user
+  end
+
+  private
+
+    attr_reader :headers
+
+    def user
+      if decoded_auth_token && Account.exists?(decoded_auth_token[:account_id])
+        Current.account = Account.find decoded_auth_token[:account_id]
+
+        Current.account.switch!
+
+        return User.find decoded_auth_token[:user_id] if User.exists? decoded_auth_token[:user_id]
+      end
+
+      errors.add(:token, 'Invalid token') && nil
```

Comment:
Acá lo podríamos hacer con un símbolo y después lo cargamos en el i18n, algo así:
```suggestion
      errors.add(:token, :invalid) && nil
```

Aunque no se si funciona igual que en los modelos, sino i18n derecho.

Reply:
esta no pude franco, porque necesitan ser activerecord